ruby/lib/google/protobuf/message_exts.rb

```diff
@@ -25,6 +25,10 @@ def to_proto(options = {})
         self.class.encode(self, options)
       end
 
+      def to_hash
+        self.to_h
```

Comment:
Does `alias_method` work just as well here?

Reply:
`alias_method` requires defining an original method before using. 

```ruby
class A
  def a
    puts "a"
  end 

  alias_method :b, :a
end 
``` 

This works, but,

```ruby
class A
  alias_method :b, :a 

  def a
    puts "a"
  end 
end 
```  

this doesn't work.

`to_h` isn't defined in `MessageExts`(It's defined in messages classes). So we can't use `alias_method` for `to_h` here.